examples/acoustic_example.py

```diff
@@ -135,3 +141,6 @@ def damp_boundary(damp, h, nbpml):
 born_op = BornOperator(dm, m, src, damp, rec, t_order, spc_order)
 print("Applying")
 born_op.apply()
+
+# remove memmap file
+DenseData.remove_memmap_file()
```

Comment:
This wouldn't be needed if you place the deletion code inside the object destructor